themes/classic/_dev/css/components/newsletter.scss

```diff
@@ -1,6 +1,7 @@
 .block_newsletter {
   @include search-box();
   float: inherit;
+  clear:both;
   width: 100%;
```

Comment:
```suggestion
  width: 100%;
  clear: both;
```

Reply:
SCSS is blocking the PR, seems like width must come before clear, and it was missing a space after `: `

Reply:
done